pymc3/distributions/transforms.py

```diff
@@ -400,7 +400,7 @@ def jacobian_det(self, y):
 
 class SumTo1(Transform):
     """
-    Transforms K dimensional simplex space (values in [0,1] and sum to 1) to K - 1 vector of values in [0,1]
+    Transforms K - 1 dimensional simplex space (k values in [0,1] and sum to 1) to K - 1 vector of values in [0,1]
```

Comment:
Minor grammar corrections -  should be:

-  "and that sum to 1)"  [insert "that"]
-  "to a K - 1 vector..." [insert "a"]